i18n/en.json

```diff
@@ -4502,6 +4494,10 @@
     "id": "model.channel.is_valid.id.app_error",
     "translation": "Invalid Id."
   },
+  {
+    "id": "model.channel.is_valid.name.app_error",
+    "translation": "Invalid name. User ids are not permitted in channel name for non-direct message channels."
```

Comment:
Recommend using what we had prior (includes the word "channel" before "name"):  "Invalid channel name. User ids are not permitted in channel name for non-direct message channels."

Reply:
@wiersgallak I have updated the wording here now